### DISCUSSION TITLE
Should android minification on release builds be enabled by default?

### DISCUSSION THREAD

**Discussion by MechMel** [ionic-team/capacitor#7430] [Ideas]
Is there a reason minification of Android release builds is not enable buy default? https://developer.android.com/build/shrink-code#groovy-build.gradle

**Comment by MechMel** [comment]
More than halved my bundle size by enabling this.

**Comment by jcesarmobile** [comment]
Since plugins execution involves reflection it can cause problems.
Nowadays we have added proguard rules to not obfuscate Capacitor required classes, but could still cause issues with 3rd party plugins.
